stm32/unittest/scripts/tst_api_demod_setup

```diff
@@ -85,6 +85,66 @@ case "${TEST_OPT}" in
 		> ref_gen.log 2>&1
 	;;
 
+    700E_plain_test )
+    	# Config is <mode>, <teswtframes> 
+    	echo "81000010" > stm_cfg.txt
+	#
+        # Copy N frames of a raw audio file to stm_in.raw. 
+        dd bs=1280 count=100 if=../../../../raw/hts1.raw of=spch_in.raw \
+		> setup.log 2>&1
+	freedv_tx 700E spch_in.raw stm_in.raw --testframes --txbpf 1 \
+		>> setup.log 2>&1
+	#
+        # Reference
+        freedv_rx 700E stm_in.raw ref_demod.raw -v --testframes \
+		> ref_gen.log 2>&1
+	;;
+
+    700E_AWGN_test )
+    	# Config is <mode>, <teswtframes> 
+    	echo "81000010" > stm_cfg.txt
+	#
+        # Copy N frames of a raw audio file to stm_in.raw. 
+        dd bs=1280 count=96 if=../../../../raw/hts1.raw of=spch_in.raw \
+		> setup.log 2>&1
+	freedv_tx 700E spch_in.raw mod_bits.raw --testframes --txbpf 1 \
+		  >> setup.log 2>&1
+	cohpsk_ch mod_bits.raw stm_in.raw -20 --Fs 8000 -f -5  2>&1 | tee setup.log
+	
+        # Reference: - When the OFDM modem initially syncs, it often
+	#              has residual freq offset that causes abnormally
+	#              high LDPC decoder bit errors forthe first few
+	#              seconds.  This leads to a high coded BER being
+	#              reported for short duration tests.  This
+	#              settles down after a few seconds, and we get
+	#              the expected coded BER when averaging over
+	#              longer periods (e.g. 60s). However this
+	#              particular test is necessarily short due to the
+	#              slow speed of the semihosting system.  It is
+	#              therefore sufficient to check that the
+	#              performance is similar to the x86 C verison,
+	#              rather than expecting a low coded BER for a
+	#              short run.
+	
```

Comment:
:point_up: this text refers to 700D, 700E can sync much faster - so we should see good coded BER results here

Reply:
Yep:

```
26: Breakpoint 1, Reset_Handler () at /home/mooneer/codec2/stm32/unittest/src/startup_stm32f4xx.s:119
26: 119	  bl  .
26: shutdown command invoked
26: shutdown command invoked
26: Remote connection closed
26: Check reference decode
26: REF   uncoded BER: 0.075200 coded BER: 0.018300
26: OK
26: Check target decode
26: TARGET uncoded BER: 0.074000 coded BER: 0.019100
26: OK
26: 
26: Test PASSED
26: 
26: Test tst_api_demod_700E_AWGN_test PASSED
1/1 Test #26: tst_api_demod_700E_AWGN_test .....   Passed   92.90 sec
```